scripts/travis-extract-l10n.sh

```diff
@@ -0,0 +1,133 @@
+#! /bin/bash
+# This script is supposed to be running automatically via travis cronjobs.
+# It's also possible to be run manually if necessary.
+#
+# This script will do the following:
+#   - prepare git credentials for pull request push
+#   - create a new branch (l10n-extract-2017-08-24-0b3bcaf2ca)
+#   - Update your code
+#   - Extract new strings and push to the .po files
+#
+# If you're running the script manually please make sure to expose
+# the following variables to the environment:
+#   - GITHUB_TOKEN (to the github token of addons-robot,
+#                   talk to tofumatt or cgrebs)
+#   - TRAVIS_REPO_SLUG="mozilla/addons-server"
+#   - TRAVIS_BRANCH="master"
+
+set -o errexit -o nounset
+
+REV=$(git rev-parse --short HEAD)
+MESSAGE="Extracted l10n messages from $(date -u --iso-8601=date) at $REV"
+BRANCH_NAME="l10n-extract-$(date -u --iso-8601=date)-$REV"
+ROBOT_EMAIL="addons-dev-automation+github@mozilla.com"
+ROBOT_NAME="Mozilla Add-ons Robot"
+
+# gettext flags
+CLEAN_FLAGS="--no-obsolete --width=200 --no-location"
+MERGE_FLAGS="--update --width=200 --backup=none"
+UNIQ_FLAGS="--width=200"
+DEBUG_LOCALES="dbl dbr"
+
+function init_environment {
+    git remote set-url --push origin "https://addons-robot:$GITHUB_TOKEN@github.com/$TRAVIS_REPO_SLUG/"
```

Comment:
Any way to avoid that, or at least change the remote name, so that we can run the script locally?

Reply:
oops, I didn't push a commit. This was already fixed in abc13bbc9f3edecc8c114a3328916035063fa25c